src/diffusers/schedulers/scheduling_ddim.py

```diff
@@ -113,6 +113,7 @@ class DDIMScheduler(SchedulerMixin, ConfigMixin):
     """
 
     _compatibles = _COMPATIBLE_STABLE_DIFFUSION_SCHEDULERS.copy()
+    _deprecated_kwargs = ["predict_epsilon"]
```

Comment:
@pcuenca what do you think about this logic? It has another advantage in that I can now easily write a test for all models and schedulers that verifies that `kwarg` should only be in __init__ if this list is > 0

Reply:
I like it! We don't have to introduce workarounds, it gives us a clean path to deprecate or replace arguments and is testable. Very cool.